### ISSUE TITLE
Displaying a table

### ISSUE THREAD

**Issue by deronbrowne** [deronbrowne/Bookkeeper-Buddy#7]
https://github.com/deronbrowne/Bookkeeper-Buddy/blob/e6f8d816aec56b4a5fdf33b5ffe848e9421199ce/Inventory.py#L51

@vkee I'm not sure what's the best way to display a table in Python (if there even is a way to do that). I'd like to be able to do that for the 'view_inventory' method in the inventory class.

**Comment by vkee** [comment]
With a GUI or in terminal?

**Comment by deronbrowne** [comment]
In the terminal. Instead of just text, I'd like to lay it out a little better. Can it not do that? Does it even make sense to try to do that at this point, or should I leave that for when I'm trying to do the interface/GUI?

**Comment by vkee** [comment]
Yes you can format the text. It depends on how nice you want it to look. It shouldn't be too hard to do. I can help in a few hours, let me know.

**Comment by vkee** [comment]
How do these look? https://pypi.python.org/pypi/terminaltables or https://github.com/nirum/tableprint
